lib/widgets/theme.dart

```diff
@@ -316,7 +331,10 @@ class DesignVariables extends ThemeExtension<DesignVariables> {
       borderBar: borderBar ?? this.borderBar,
       contextMenuCancelText: contextMenuCancelText ?? this.contextMenuCancelText,
       contextMenuItemBg: contextMenuItemBg ?? this.contextMenuItemBg,
+      contextMenuItemLabel: contextMenuItemLabel ?? this.contextMenuItemLabel,
+      contextMenuItemMeta: contextMenuItemMeta ?? this.contextMenuItemMeta,
       contextMenuItemText: contextMenuItemText ?? this.contextMenuItemBg,
```

Comment:
Oh wow, something's wrong in the code that was already here before your changes:

```suggestion
      contextMenuItemText: contextMenuItemText ?? this.contextMenuItemText,
```

It would be helpful if you could add a small commit fixing that and a similar instance in `lerp` below.

Reply:
Oh, didn't even notice this